src/charm.py

```diff
@@ -269,21 +273,22 @@ def _get_srsenb_command(self) -> str:
                 f'--enb_files.rr_config={CONFIG_PATHS["rr"]}',
                 f'--enb_files.sib_config={CONFIG_PATHS["sib"]}',
                 f'--enb_files.drb_config={CONFIG_PATHS["drb"]}',
+                CONFIG_PATHS["enb"],
                 f'--rf.device_name={self.config.get("enb-rf-device-name")}',
                 f'--rf.device_args={self.config.get("enb-rf-device-args")}',
             )
         )
         return " ".join(srsenb_command)
 
-    def _get_srsue_command(self) -> str:
+    def _get_srsue_command(self, ue_usim_imsi: str, ue_usim_k: str, ue_usim_opc: str) -> str:
         """Returns srs ue command."""
         srsue_command = [SRS_UE_BINARY]
-        if self._get_peer_data("ue_usim_imsi"):
+        if ue_usim_imsi:
```

Comment:
Why are we only checking for the `ue_usim_imsi`?

Reply:
Actually it's not needed anymore so I removed it